File: Homework/Homework/CalculatorProblem.cs
namespace HomeWork
{
    public class CalculatorProblem
    {
        public int Val1 { get; }
        public int Val2 { get; }
        public CalculatorOperation Operation { get; }

        public CalculatorProblem(int val1, CalculatorOperation operation, int val2)
        {
            Val1 = val1;
            Operation = operation;
            Val2 = val2;
        }

        public CalculatorProblem()
        { }
    }
}

File: Homework4/CalculatorProblem.fs
namespace Homework4

type CalculatorProblem (val1: int, operation: CalculatorOperation, val2: int) =
    member this.Val1 = val1
    member this.Val2 = val2
    member this.Operation = operation
    new() = CalculatorProblem(0, CalculatorOperation.Plus, 0)
